frontend/src/components/Form.js

```diff
@@ -41,11 +41,45 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+/**
+ * Validates from inputs on submission and alerts user on error
+ * @param {String} role
+ * @param {String} productArea
+ * @param {Number} duration
+ * @param {Number} timeAvailableUntilMilliseconds
+ * @param {Number} currentTimeMilliseconds
+ * @return {Boolean} true or false based on validity of inputs
+ */
+export function validateFormInputs(
+    role,
+    productArea,
+    duration,
+    timeAvailableUntilMilliseconds,
+    currentTimeMilliseconds,
+) {
+  const durationMilliseconds = duration * 60000;
+
+  if (isNaN(timeAvailableUntilMilliseconds)) {
+    alert('Please select a valid date.');
+    return false;
+  } else if (currentTimeMilliseconds + durationMilliseconds >=
+    timeAvailableUntilMilliseconds) {
+    // Check if a meeting is possible with the provided inputs
+    alert('Please select an larger time availability window');
+    return false;
+  } else if (productArea === '' || role === '') {
+    alert('Please select options for all form fields to submit!');
+    return false;
+  }
+  return true;
+}
+
 /**
  * Create form component with time and match preference inputs
+ * @param {Object} props
  * @return {Form} Form component
  */
-export default function Form() {
+export default function Form(props) {
```

Comment:
Any reason why we need to add `props` to the arguments?

Reply:
This is used for communication purposes so that the child component can communicate with the parent component in the future.